### PR TITLE
ShowHiddenChannels: better ui, alternative display mode

### PR DESCRIPTION
**Pull request by Nuckyz**

### CODE REVIEW

**Comment by Nuckyz** [comment]
Found an issue that I will fix later

**Comment by Nuckyz** [comment]
Ready

File: src/plugins/showHiddenChannels.tsx [hunk 1]
```diff
@@ -23,125 +23,177 @@ import { Flex } from "@components/Flex";
...
 export default definePlugin({
     name: "ShowHiddenChannels",
-    description: "Show hidden channels",
-    authors: [Devs.BigDuck, Devs.AverageReactEnjoyer, Devs.D3SOX, Devs.Ven],
+    description: "Show channels that you do not have access to view.",
+    authors: [Devs.BigDuck, Devs.AverageReactEnjoyer, Devs.D3SOX, Devs.Ven, Devs.Nuckyz, Devs.Nickyux],
     options: {
         hideUnreads: {
-            description: "Hide unreads",
+            description: "Hide Unreads",
```
Comment (Vendicated) [inline src/plugins/showHiddenChannels.tsx:56]: whats the point of this setting btw, why would you ever want to see unreads in a hidden channel

**Review by Vendicated — APPROVED** [review]
have you tested that Message Logger still works?

File: src/plugins/showHiddenChannels.tsx (hunk 1 above)
Comment (Nuckyz) [reply src/plugins/showHiddenChannels.tsx:56]: dunno, I didn't implement it

**Comment by Nuckyz** [comment]
> have you tested that Message Logger still works?

not exactly since they were fairly simple changes

**Comment by Delphox** [comment]
Hey something on this PR broke the Hide Muted Channels hiding muted hidden channels.

Just tested on Vencord 8a43e9b and it still works:

https://user-images.githubusercontent.com/5883538/214331072-4a0ec48e-8d58-4dc5-84cf-154cb1129cb4.mp4

However on Vencord 75050e7 and above it doesn't:

https://user-images.githubusercontent.com/5883538/214331439-d81a1a7b-e143-407a-899a-46f401b8c488.mp4

**Comment by Nuckyz** [comment]
Fixed in

> Hey something on this PR broke the Hide Muted Channels hiding muted hidden channels.
> 
> Just tested on Vencord [8a43e9b](https://github.com/Vendicated/Vencord/commit/8a43e9b25f97cda19a5dc85020152020873c6a8e) and it still works:
> 
>  thorium_uJTBWscy6K.mp4 
> However on Vencord [75050e7](https://github.com/Vendicated/Vencord/commit/75050e74ca01856f9b48867d7945e2b7263b42ae) and above it doesn't:
> 
>  thorium_M64HvKHB59.mp4

Fixed in #454

**Pull request merged**